src/services/weatherService.js

```diff
@@ -0,0 +1,53 @@
+import { getDayOfTheWeekName, getDateTime } from '../utils/dateUtils';
+import { API_KEY } from '../secrets';
+import { kelvinToCelsius } from '../utils/weatherUtils';
+
+const convertToCelsiusAndFloor = (t) => Math.floor(kelvinToCelsius(t));
```

Comment:
How about omitting parentheses if there's only one param ? ;)

Reply:
Agree. I should also reconfigure the linter.